aiohttp/abc.py

```diff
@@ -120,7 +120,7 @@ def request(self) -> Request:
         return self._request
 
     @abstractmethod
-    def __await__(self) -> Generator[Any, None, StreamResponse]:
+    def __await__(self) -> Generator[None, None, StreamResponse]:
```

Comment:
```suggestion
    def __await__(self) -> Generator[asyncio.Future[Any], None, StreamResponse]:
```
I guess, technically, asyncio.Task never sends non-None values into coroutines but respects to receive `asyncio.Future[Any]`

Reply:
Typeshed at the moment doesn't seem to match this. We can update this later if typeshed gets updated.